grpc/send_incoming_call_notification.go

```diff
@@ -0,0 +1,54 @@
+package grpc
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"log"
+
+	pb "github.com/PretendoNetwork/grpc-go/friends"
+	nex "github.com/PretendoNetwork/nex-go"
+	friends_wiiu_types "github.com/PretendoNetwork/nex-protocols-go/friends-wiiu/types"
+	nintendo_notifications "github.com/PretendoNetwork/nex-protocols-go/nintendo-notifications"
+	nintendo_notifications_types "github.com/PretendoNetwork/nex-protocols-go/nintendo-notifications/types"
+	"github.com/PretendoNetwork/wiiu-chat-secure/globals"
+	"google.golang.org/grpc/metadata"
+)
+
+func SendIncomingCallNotification(caller uint32, target uint32) {
+	ctx := metadata.NewOutgoingContext(context.Background(), globals.GRPCFriendsCommonMetadata)
+
+	presence := friends_wiiu_types.NewNintendoPresenceV2()
+
+	presence.ChangedFlags = 0x1FF
+	presence.Online = true
+	presence.GameKey = friends_wiiu_types.NewGameKey()
+	presence.Unknown2 = 0x65
+	presence.GameServerID = 0x1005A000
+	presence.PID = 4 // This is not a PID.
+	presence.GatheringID = caller
+
+	// The application data here is the PID, repeated 4 times. Why, you ask? Who knows!
```

Comment:
Is this related to the 4 value above?

Reply:
I hadn't tested this, but in fact, it is! It's the amount of times the PID gets repeated in the application data. Don't know why.

This also resolves the other conversation.